ray-operator/controllers/common/status.go

```diff
@@ -0,0 +1,45 @@
+package common
```

Comment:
it seems the file name here is more `statusUtils.go` than `status.go`

Reply:
Make sense. I will move these methods to utils.go.